.github/copilot-instructions.md

```diff
@@ -134,6 +136,7 @@ When working with CSS if() functions, always follow the official WCAG specificat
 3. **Always** run the full test suite including browser validation
 4. **Always** update documentation when changing public APIs
 5. **Always** bear in mind that the developer's main job is to read, not write, code. Therefore, avoid unnecessary complexity, abbreviations and short forms of parameters, for example in CLI usage.
-6. **Consider** performance impact on large stylesheets and DOM trees
+6. **Always** try to prevent setting up seperate fixtures by each output (polyfill, PostCSS plugin, lightniningcss plugin) and instead use the same fixture for all of them, if possible. In case of any differences try to change the fixture in a way that it can be used for all of them, e.g. changing the color values in the fixture so that it would be the same output by all of them.
```

Comment:
```suggestion
6. **Always** try to prevent setting up seperate fixtures by each output (polyfill, PostCSS plugin, Lightning CSS plugin) and instead use the same fixture for all of them, if possible. In case of any differences try to change the fixture in a way that it can be used for all of them, e.g. changing the color values in the fixture so that it would be the same output by all of them.
```